e2e/tests/workspace/workspace-duplicate.spec.ts

```diff
@@ -76,4 +77,85 @@ describe('Duplicate workspace', () => {
       await dataPage.deleteWorkspace();
     });
 
+  test('OWNER can duplicate workspace to an older CDR Version via Workspace card', async () => {
+    const workspaceCard: WorkspaceCard = await createWorkspace(page, config.defaultCdrVersionName);
+    const originalWorkspaceName = await workspaceCard.getWorkspaceName();
+
+    await workspaceCard.asElementHandle().hover();
+    // Click on Ellipsis menu "Duplicate" option.
+    await workspaceCard.selectSnowmanMenu(Option.Duplicate);
+
+    // Fill out Workspace Name should be just enough for successful duplication
+    const workspacesPage = new WorkspacesPage(page);
+    await (await workspacesPage.getWorkspaceNameTextbox()).clear();
+    const duplicateWorkspaceName = await workspacesPage.fillOutWorkspaceName();
+
+    // change CDR Version
+    await workspacesPage.selectCdrVersion(config.altCdrVersionName);
+
+    const finishButton = await workspacesPage.getDuplicateWorkspaceButton();
+    await finishButton.waitUntilEnabled();
+    await workspacesPage.clickCreateFinishButton(finishButton);
+
+    // Duplicate workspace Data page is loaded.
+    const dataPage = new WorkspaceDataPage(page);
+    await dataPage.waitForLoad();
+    expect(page.url()).toContain(duplicateWorkspaceName.replace(/-/g, '')); // Remove dash from workspace name
+
+    // Delete duplicate workspace via Workspace card in Your Workspaces page.
+    await Navigation.navMenu(page, NavLink.YOUR_WORKSPACES);
+    await workspacesPage.waitForLoad();
+
+    await WorkspaceCard.deleteWorkspace(page, duplicateWorkspaceName);
+
+    // Verify Delete action was successful.
+    expect(await WorkspaceCard.findCard(page, duplicateWorkspaceName)).toBeFalsy();
+
+    // Delete original workspace via Workspace card
+    await WorkspaceCard.deleteWorkspace(page, originalWorkspaceName);
+    expect(await WorkspaceCard.findCard(page, originalWorkspaceName)).toBeFalsy();
+  });
+
+  test('OWNER can duplicate workspace to a newer CDR Version via Workspace card', async () => {
+    const workspaceCard: WorkspaceCard = await createWorkspace(page, config.altCdrVersionName);
+    const originalWorkspaceName = await workspaceCard.getWorkspaceName();
+
+    await workspaceCard.asElementHandle().hover();
+    // Click on Ellipsis menu "Duplicate" option.
+    await workspaceCard.selectSnowmanMenu(Option.Duplicate);
+
+    // Fill out Workspace Name should be just enough for successful duplication
+    const workspacesPage = new WorkspacesPage(page);
+    await (await workspacesPage.getWorkspaceNameTextbox()).clear();
+    const duplicateWorkspaceName = await workspacesPage.fillOutWorkspaceName();
+
+    // change CDR Version
+    await workspacesPage.selectCdrVersion(config.defaultCdrVersionName);
+
+    const upgradeMessage = await workspacesPage.getCdrVersionUpgradeMessage();
+    expect(upgradeMessage).toContain(`You're duplicating the workspace "${originalWorkspaceName}" to upgrade from`);
```

Comment:
nit: is the full text needed for the test? Would the `originalWorkspaceName` be enough?